.github/workflows/conventional-commits.yml

```diff
@@ -0,0 +1,19 @@
+name: Conventional Commits
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Do we want to run it for all branches?

Reply:
@Youssef1313 Why would we not run it on all?

Reply:
@morning4coffe-dev Exactly my question :)

Currently this only run against master, but not in feature branches

Reply:
@morning4coffe-dev My bad :) I wanted to ask why aren't we running it in our other repos on all branches?

Reply:
cc @jeromelaban

Reply:
It's probably best to have it running as a job in azure devops, like this one: https://github.com/unoplatform/uno/blob/master/build/ci/.azure-devops-commitsar.yml